docs/hub/models-libraries.md

```diff
@@ -4,25 +4,25 @@ The Hub has support for dozens of libraries in the Open Source ecosystem. Thanks
+| [docTR](https://github.com/mindee/doctr)                                    | Models and datasets for OCR-related tasks in PyTorch & TensorFlow                    | ✅             |       ✅ | ✅                 | ❌           |
+| [ESPnet](https://github.com/espnet/espnet)                                  | End-to-end speech processing toolkit (e.g. TTS)                                      | ✅             |       ✅ | ✅                 | ❌           |
+| [Flair](https://github.com/flairNLP/flair)                                  | Very simple framework for state-of-the-art NLP.                                      | ✅             |       ✅ | ✅                 | ❌           |
+| [NeMo](https://github.com/NVIDIA/NeMo)                                      | Conversational AI toolkit built for researchers                                      | ✅             |       ✅ | ✅                 | ❌           |
+| [Pyannote](https://github.com/pyannote/pyannote-audio)                      | Neural building blocks for speaker diarization.                                      | ❌             |       ❌ | ✅                 | ❌           |
+| [PyCTCDecode](https://github.com/kensho-technologies/pyctcdecode)           | Language model supported CTC decoding for speech recognition                         | ❌             |       ❌ | ✅                 | ❌           |
+| [Sentence Transformers](https://github.com/UKPLab/sentence-transformers)    | Compute dense vector representations for sentences, paragraphs, and images.          | ✅             |       ✅ | ✅                 | ✅           |
+| [spaCy](https://github.com/explosion/spaCy)                                 | Advanced Natural Language Processing in Python and Cython.                           | ✅             |       ✅ | ✅                 | ✅           |
+| [Speechbrain](https://speechbrain.github.io/)                               | A PyTorch Powered Speech Toolkit.                                                    | ✅             |       ✅ | ✅                 | ❌           |
+| [TensorFlowTTS](https://github.com/TensorSpeech/TensorFlowTTS)              | Real-time state-of-the-art speech synthesis architectures.                           | ❌             |       ❌ | ✅                 | ❌           |
+| [Timm](https://github.com/rwightman/pytorch-image-models)                   | Collection of image models, scripts, pretrained weights, etc.                        | ❌             |       ❌ | ✅                 | ❌           |
+| [Stable-Baselines3](https://github.com/DLR-RM/stable-baselines3)            | Set of reliable implementations of deep reinforcement learning algorithms in PyTorch | ❌             |       ✅ | ✅                 | ✅           |
+| [fastai](https://github.com/fastai/fastai)                                  | Library to train fast and accurate models with state-of-the-art outputs.             | ✅             | ✅       | ✅                 | ✅           |
```

Comment:
We can keep alphabetical order here :smile:

Reply:
Done 🤗. I also moved sb3 to follow alphabetical order.